src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static checkingIsAlive() {
+    const aliveAnimals = this.alive.filter(animal => animal.health > 0);
+
+    this.alive.length = 0;
+    this.alive = [...aliveAnimals];
```

Comment:
Changing array length and creating a copy is redundant here
```suggestion
    this.alive = this.alive.filter(animal => animal.health > 0);
```